dds/DCPS/DataReaderImpl.h

```diff
@@ -88,6 +88,7 @@ class OpenDDS_Dcps_Export WriterStats {
   WriterStats(
     int amount = 0,
     DataCollector<double>::OnFull type = DataCollector<double>::KeepOldest);
+  WriterStats(const WriterStats&);
```

Comment:
These were warning cleanup for my compiler on Ubuntu (gcc 9.2.1), which said that the default constexpr copy constructors were deprecated in the case where an operator= had been created.

Reply:
With c++11 enabled you can use default, see some earlier changes in ace

Reply:
Yeah, wanted to check what cppreference said about it:

The generation of the implicitly-defined copy constructor is deprecated if T has a user-defined destructor or user-defined copy assignment operator.  (since C++11)

Reply:
> The generation of the implicitly-defined copy constructor is deprecated if T has a user-defined destructor or user-defined copy assignment operator. (since C++11)

If that's the case you can get rid of the of assignment operator on `Time Duration` at least.

Reply:
Also `WriterStats`, doesn't look like it meets the definition that you gave, did you still get warnings about it?

Reply:
Yeah, I had warnings about it, though I may have misunderstood what it was telling me. It was probably complaining about the implicit copy constructor for Stats (its member) which also had this issue because of the assignment operator.